test/index-static.test.js

```diff
@@ -1,12 +1,24 @@
 var should = require('should');
 
-require('node-jsx').install();
-
-var indexStatic = require('../lib/index-static.jsx');
+var indexStaticWatcher = require('../lib/index-static-watcher').create();
 
 describe('index-static', function() {
+  var indexStatic;
+
+  this.timeout(10000);
```

Comment:
Yep! It takes longer than mocha's default of 2000ms on my 5-yr old laptop, so i wanted to make sure it had plenty of time...